data/trie/patriciaMerkleTrie_test.go

```diff
@@ -538,19 +553,40 @@ func BenchmarkPatriciaMerkleTrie_RootHashAfterChanging30000Nodes(b *testing.B) {
 		key := hsh.Compute(strconv.Itoa(i))
 		value := append(key, []byte(strconv.Itoa(i))...)
 
-		tr.Update(key, value)
+		_ = tr.Update(key, value)
 		values[i] = key
 	}
-	tr.Commit()
+	_ = tr.Commit()
 
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
 		b.StopTimer()
 		for j := 0; j < nrOfValuesToModify; j++ {
-			tr.Update(values[j], values[j])
+			_ = tr.Update(values[j], values[j])
 		}
 		b.StartTimer()
-		tr.Root()
+		_, _ = tr.Root()
+	}
+}
+
+func BenchmarkPatriciaMerkleTrie_Cloning10000ValuesTrie(b *testing.B) {
+	tr := emptyTrie()
+	hsh := keccak.Keccak{}
+
+	nrValuesInTrie := 10000
+	values := make([][]byte, nrValuesInTrie)
+
+	for i := 0; i < nrValuesInTrie; i++ {
+		key := hsh.Compute(strconv.Itoa(i))
+		value := append(key, []byte(strconv.Itoa(i))...)
+
+		_ = tr.Update(key, value)
+		values[i] = key
```

Comment:
I think you can remove this line, and also line 577.

Reply:
done